torch/csrc/autograd/functions/basic_ops.cpp

```diff
@@ -27,4 +27,25 @@ auto DelayedError::apply(variable_list&& inputs) -> variable_list {
   });
 }
 
+auto UndefinedGrad::apply(variable_list&& inputs) -> variable_list {
+  tensor_list outputs;
```

Comment:
Given that these Nodes are used for debugging only, maybe the simplest here would be to `clone()` in the forward. That way you can ignore all the view/version counter questions.